docs/integration/code-adaptions.mdx

```diff
@@ -80,19 +91,27 @@ private void Start()
 <TabItem value="js" label="JavaScript">
 
 ```js
-alert("TODO");
+const ArcadeClient = new ArcadeClientSDK();
+
+// The client constructor will handle fetching the
+// token for the player
+
+// If "", wait until the client has fetched the token
+while (ArcadeClient.sessionToken == "") {
+    // sleep for 10ms so the CPU is not sent to 100%
+}
```

Comment:
here is code missing that takes the server address and connects to that address (or starts free-to-play mode).
For Unity this is automatically handled in the AutoConnect behavior

Reply:
done